tests/helpers/calc/test_calc.py

```diff
@@ -103,13 +104,15 @@ def test_calc_functions_multiple_arguments():
     """Tests calc.py handling functions with multiple arguments correctly"""
     def h1(x): return x
 
-    def h2(x, y): return x + y
+    def h2(x, y): return x * y
 
-    def h3(x, y, z): return x + y + z
+    def h3(x, y, z): return x * y * z
 
     assert evaluator("h(2)", {}, {"h": h1}, {})[0] == 2.0
-    assert evaluator("h(1, 2)", {}, {"h": h2}, {})[0] == 3.0
-    assert evaluator("h(1, 2, 3)", {}, {"h": h3}, {})[0] == 6.0
+    assert evaluator("h(2, 3)", {}, {"h": h2}, {})[0] == 6.0
+    assert evaluator("h(2, 3, 4)", {}, {"h": h3}, {})[0] == 24.0
+    assert equal_as_arrays(evaluator("h(2, [1, 2, 3])", {}, {"h": h2})[0],
```

Comment:
By the way: Line 114 would have failed before this PR, but succeeds now. I changed the `h`s to be multiplication so that in `h2(x, y)`, the arguments `x` and `y` could be number and MathArray. (which would not work for addition).

Oh. Well I guess I could have used addition and `0`. Oh well.